readme.md

```diff
@@ -118,6 +118,9 @@ If that doesn't work, [try this line](https://github.com/georgestephanis/applica
 
     RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
 
+You can also try:
+    CGIPassAuth On
```

Comment:
I would add a new line here, otherwise there is a risk that it might not format correctly